thundra/plugins/invocation/invocation_support.py

```diff
@@ -38,10 +39,18 @@ def get_tags():
 def remove_tag(key):
     return _user_invocation_tags.pop(key, None)
 
-
 def clear():
+    global _user_error
     _invocation_tags.clear()
     _user_invocation_tags.clear()
+    _user_error = None
```

Comment:
This is cleared on after invocation phase of invocation plugin. But user error is also read at after invocation phase of trace plugin. As far as I know, after invocation phase of invocation plugin is called before after invocation phase of trace plugin. So this means that even though there is error, trace plugin will see user error as empty always.